### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by gillstrom**
* Minor tweaks
* Allow changing name of cookie
* Add attribute to set a secure cookie

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -66,6 +73,12 @@ Type: `function`
 
 Function that runs on button click.
 
+### secure
+
+Type: `boolean`
```
Comment (kevva) [inline readme.md]: Default: `false`?

**Comment by kevva** [comment]
Might as well do the following too:

* Use tabs in `package.json`
* Add `.npmrc` with `package-lock=false` and add `yarn.lock` to `.gitignore`
* Remove `"esnext": true` from `package.json` (`true` by default)

**Review by kevva — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (gillstrom) [reply readme.md]: Fixed

**Comment by gillstrom** [comment]
Fixed the meta tweaks.

**Review by kevva — APPROVED** [review]

**Comment by kevva** [comment]
LGTM 👍

**Pull request merged**